examples/scenarioGaussMarkovRandomWalk.py

```diff
@@ -0,0 +1,255 @@
+#
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+
+r"""
+Overview
+---------
+
+This script demonstrates the difference between bounded Gauss-Markov random walk and pure Gaussian noise by simulating
+two IMU sensors with different configurations. The simulation shows how the Gauss-Markov process can be configured
+either to maintain bounded random walk behavior or to generate pure Gaussian noise. The script sets up a spacecraft
+in an elliptical Earth orbit and attaches two IMU sensors with different noise configurations.
+
+The script is found in the folder ``basilisk/examples`` and executed by using::
+
+    python3 scenarioGaussMarkovRandomWalk.py
+
+When the simulation completes a plot is shown for the IMU sensor measurements over time.
+
+Simulation Scenario Setup Details
+----------------------------------
+The simulation layout is shown in the following illustration. A single simulation process is created which
+contains both the spacecraft simulation module and two IMU sensor modules. Each IMU sensor has different
+configurations for the process noise (P Matrix), state propagation (A Matrix), and walk bounds.
+
+The dynamics simulation is setup using a :ref:`Spacecraft` module in an elliptical Earth orbit. The IMU sensors
+are configured with different parameters:
+
+- IMU 1 demonstrates bounded random walk behavior:
+    - Uses non-zero A Matrix for state propagation (-0.1 on diagonal)
+    - Has positive walk bounds (±3.0 rad/s by default)
+    - Shows mean-reverting behavior characteristic of Gauss-Markov process
+    - Uses process noise level of 0.5 by default
+
+- IMU 2 demonstrates pure Gaussian noise:
+    - Uses zero A Matrix (set after initialization)
+    - Has negative walk bounds to disable random walk
+    - Shows independent noise samples with no temporal correlation
+    - Uses the same process noise level as IMU 1
+
+Both IMUs use the same process noise level (P Matrix) to ensure comparable noise magnitudes.
+
+Illustration of Simulation Results
+-----------------------------------
+
+The following plot shows the angular velocity measurements from both IMU sensors over a 10-minute period:
+
+.. image:: /_images/Scenarios/scenarioGaussMarkovRandomWalk.svg
```

Comment:
I'm not getting the expected figure??  When I run the scenario manually I get the expected figure, but the figure saved off by running the test file doesn't look right?

![scenarioGaussMarkovRandomWalk](https://github.com/user-attachments/assets/6e039c8c-bb7b-4bab-81cb-1bd45779055b)

Reply:
Strange, I'll debug that, I was getting the right one from the unit tests

Reply:
I probably messed something small up in the rebase is my hunch and didn't catch it